workflow/sample_files/base-battery-scheduled-power-outage.xml

```diff
@@ -25,6 +25,7 @@
           <EndMonth>7</EndMonth>
           <EndDayOfMonth>31</EndDayOfMonth>
           <EndHourOfDay>14</EndHourOfDay>
+          <NaturalVentilation>regular schedule</NaturalVentilation>
```

Comment:
Ideally these files wouldn't change.

Reply:
See response above.

(I suppose we could relax `schedules_unavailable_period_window_natvent_availabilities ` being required if the other 2 arguments are initialized. We could also relax that its elements are all valid ventilation types, e.g., allow `,,regular schedule,,,always unavailable,,,,`.)

Reply:
Yes, I'm in favor of allowing this argument to be optional. If there are, say, 3 comma-separated values provided to the other two arguments, It'd be nice if this argument could either be `"foo,foo,foo"` or `",,foo"` or even just empty `""`.